src/Illuminate/Validation/Concerns/FormatsMessages.php

```diff
@@ -336,6 +336,10 @@ public function getDisplayableValue($attribute, $value)
             return $value ? 'true' : 'false';
         }
 
+        if (is_null($value)) {
+            return 'null';
+        }
+
```

Comment:
This is the part I'm not 100% sure about if it'll be expected by people or not. See point 3 in the PR description.

Reply:
It may be best to change this to "empty"? `null` is a very programmer term and may not be understood by end-users using an application.

Reply:
Done!

Reply:
I totally agree about null being too programmery and "empty" being better for end users, but I wonder whether it might confuse devs given "empty" also has a meaning in PHP which isn't being used here. Perhaps "blank", "missing", or "unfilled" might be a more neutral term? Or maybe I'm overthinking it.